pkg/console/operator/sync_v400.go

```diff
@@ -246,7 +252,7 @@ func (co *consoleOperator) SyncDeployment(
 	recorder events.Recorder) (consoleDeployment *appsv1.Deployment, changed bool, reason string, err error) {
 
 	updatedOperatorConfig := operatorConfig.DeepCopy()
-	requiredDeployment := deploymentsub.DefaultDeployment(operatorConfig, cm, serviceCAConfigMap, defaultIngressCertConfigMap, trustedCAConfigMap, sec, proxyConfig, infrastructureConfig, canMountCustomLogo)
+	requiredDeployment := deploymentsub.DefaultDeployment(operatorConfig, cm, clusterCAConfigMaps, serviceCAConfigMap, defaultIngressCertConfigMap, trustedCAConfigMap, sec, proxyConfig, infrastructureConfig, canMountCustomLogo)
```

Comment:
Not related to this change, but we will need to review and refactor this approach since adding parameters is not a great long term solution.

Reply:
Yeah, there are definitely some tech debt issues with this. I assume that at some point we'll want to pull the deployment sync into a separate controller which would make fixing this much easier.